pontoon/translations/views.py

```diff
@@ -22,6 +23,33 @@
 from pontoon.translations import forms
 
 
+def _send_badge_notification(user, badge, level):
+    desc = render_to_string(
+        "messaging/badge_notification.jinja",
+        {"badge": badge, "level": level, "user": user},
+    )
+    notify.send(
+        sender=user,
+        recipient=user,
+        verb="ignore",  # Triggers render of description only
+        description=desc,
```

Comment:
Please add a [category](https://github.com/mozilla/pontoon/pull/3456/commits/b1e80b02a7b2c57b7a631629e5c828f1c6643f8f) to the notification:

```suggestion
        description=desc,
        category="badge",
```